### PR TITLE
Enabled Image->PDF conversion for uploaded documents

### PR DESCRIPTION
**Pull request by markolson**

### CODE REVIEW

File: app/uploaders/claim_documentation/uploader.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class ClaimDocumentation::Uploader < VetsShrine
     validate_virus_free
     validate_max_size 20.megabytes
     validate_min_size 1.kilobytes
```
Comment (jbalboni) [inline app/uploaders/claim_documentation/uploader.rb]: Is this something I should validate on the front end, too?

File: app/uploaders/claim_documentation/uploader.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class ClaimDocumentation::Uploader < VetsShrine
...
-    validate_mime_type_inclusion %w(application/pdf) # until conversion is in
+    validate_mime_type_inclusion %w(image/jpg image/jpeg image/png application/pdf)
```
Comment (jbalboni) [inline app/uploaders/claim_documentation/uploader.rb]: Are we limited to these image types? Or is this just following what UX had mocked up?

File: app/uploaders/claim_documentation/uploader.rb (hunk 1 above)
Comment (markolson) [inline app/uploaders/claim_documentation/uploader.rb]: Hmmm yeah, if we can.

File: app/uploaders/claim_documentation/uploader.rb (hunk 2 above)
Comment (markolson) [inline app/uploaders/claim_documentation/uploader.rb]: I think we can add more types later, but these seemed good as a first crack

**Pull request merged**